### PR TITLE
cert: add description for the "info" command

### PR DESCRIPTION
**Pull request by Mikaela**

### CODE REVIEW

**Comment by DarthGandalf** [comment]
Did you test this?

**Comment by Mikaela** [comment]
I am not sure what you mean.

I didn't do more testing to this than whatever Travis did or if you mean the cert command, I ran the info and guessed that it does what I added to description.

```
2015-09-20 16:25:38+0300 < *cert> You have a certificate in: /home/users/mikaela/.znc/users/Mikaela/moddata/cert/user.pem
2015-09-20 16:25:38+0300 -- [*cert] (znc@znc.in): *cert
2015-09-20 16:25:38+0300 -- [*cert] *.znc.in (Bouncer)
2015-09-20 16:25:38+0300 -- [*cert] End of /WHOIS list.
```

**Comment by DarthGandalf** [comment]
Did you see the actual message which you added, during your tests?
It should be in output of `help`, right?

**Comment by Mikaela** [comment]
I didn't run this, if there is something wrong with the change, could you tell what it is directly?

**Comment by DarthGandalf** [comment]
Well, please test it. Testing changes is usually a good idea, especially if you're not very familiar with C++ which you're changing.
I'm sure this change didn't do what you expected it to do.

**Comment by Mikaela** [comment]
How should I change this change?

**Comment by Mikaela** [comment]
remove `,""` or make it `,"","","blah"`?

File: modules/cert.cpp [hunk 1]
```diff
@@ -46,7 +46,7 @@ class CCertMod : public CModule {
 		AddHelpCommand();
 		AddCommand("delete", static_cast<CModCommand::ModCmdFunc>(&CCertMod::Delete),
 			"", "Delete the current certificate");
-		AddCommand("info", static_cast<CModCommand::ModCmdFunc>(&CCertMod::Info));
+		AddCommand("info", static_cast<CModCommand::ModCmdFunc>(&CCertMod::Info)),"", "Shows whether or not you have a certificate";
```
Comment (jpnurmi) [inline modules/cert.cpp]: AddCommand("info", static_cast<CModCommand::ModCmdFunc>(&CCertMod::Info), "", "Shows whether or not you have a certificate");

Comment (jpnurmi) [inline modules/cert.cpp]: or just "Shows the current certificate"

Comment (Mikaela) [inline modules/cert.cpp]: Done, thanks

**Comment by jpnurmi** [comment]
https://github.com/znc/znc/commit/c8ab811a29198c085228ea7047e23833bdddc4a7

**Pull request closed without merging**